Tests/Get-BicepConfig.Tests.ps1

```diff
@@ -0,0 +1,168 @@
+BeforeAll {
```

Comment:
We are currently not mocking Get-BicepNetConfig, and that might be ok, but it makes this an integration test and not a unit test. There is one reason to at least think about this:
If we change the behaviour of Get-BicepNetConfig _without changing the end user experience of Get-BicepConfig_ it does not require new tests to be written.

Again, this might be ok and expected, but something to consider.

Reply:
Since Get-BicepConfig is a wrapper around Get-BicepNetConfig, I say we are fine with that,